src/lib/components/Footer/FooterNavContainer.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Footer.css';
+
+const FooterNavContainer = props => (
+  <nav className="p-footer__nav">
+    <ul className="p-footer__links">
+      { props.children }
+    </ul>
+  </nav>
+);
+
+FooterNavContainer.propTypes = {
+  children: PropTypes.oneOfType([
```

Comment:
this could just be `PropTypes.node`

```
 // Anything that can be rendered: numbers, strings, elements or an array
  // (or fragment) containing these types.
  optionalNode: PropTypes.node
```

From: https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
@Lukewh Good catch. I think originally I thought I wanted the children to ONLY be one or more React elements.